webpack/assets/javascripts/react_app/components/LoginPage/login-page.scss

```diff
@@ -0,0 +1,3 @@
+.login-pf {
```

Comment:
Can you call this file `LoginPage.scss`?

Reply:
done.